apps/console/src/app/pages/no-beta-access.scss

```diff
@@ -0,0 +1,9 @@
+.no-beta-access {
```

Comment:
To avoid this, could you create shared components or components on the router librarie?